### ISSUE TITLE
reverse port forward

### ISSUE THREAD

**Issue by chasebolt** [devspace-cloud/devspace#1030]
**Is your feature request related to a problem?**  
I have php xdebug inside of our containers and for developers that are using k3s/minikube it works fine. But for developers that are using remote clusters it is unable to connect back to them.

```
xdebug.remote_enable=1
xdebug.remote_connect_back=1
xdebug.remote_port=9000
xdebug.remote_autostart=1
```

**Which solution do you suggest?**  
Implement a reverse tunnel to improve portability.

```
xdebug.remote_enable=1
xdebug.remote_host=127.0.0.1
xdebug.remote_port=9000
xdebug.remote_autostart=1
```

/kind feature

**Comment by FabianKramm** [comment]
@chasebolt thanks for opening this issue! Yes that sounds like a good idea, the implementation will need us to inject a ssh server into the container with which we can do a remote portforwarding to the local computer.

**Comment by DrDonk** [comment]
Have a look at tunnel which solves this problem without SSH.
[https://github.com/omrikiei/ktunnel](url)

**Comment by paneq** [comment]
This would be great to have to support hybrid development environment where some services are deployed in K8s and one or two applications are running natively on the developer's machine.

**Comment by LukasGentele** [comment]
Implemented as part of [v4.13](https://github.com/devspace-cloud/devspace/releases/tag/v4.13.0)
